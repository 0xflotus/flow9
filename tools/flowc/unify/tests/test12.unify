α17 c= () -> void;
() -> void c= α18;
