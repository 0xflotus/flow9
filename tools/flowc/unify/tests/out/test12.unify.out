α17=() -> void
α18=() -> void
α100=void
α101=void
